test/ConsoleHelperTest.php

```diff
@@ -81,8 +81,8 @@ public function testCanColorizeMixedStrings()
         $string = "<error>NOT OK</error>\n\n<info>Usage:</info> foo";
         $colorized = $this->helper->colorize($string);
 
-        $this->assertContains("\033[31mNOT OK\033[0m", $colorized, 'Colorized error string not found');
-        $this->assertContains("\033[32mUsage:\033[0m", $colorized, 'Colorized info string not found');
+        $this->assertStringContainsString("\033[31mNOT OK\033[0m", $colorized, 'Colorized error string not found');
```

Comment:
Not important, but assertions are static methods.

Reply:
I can change it easily for all assertions - I do have it implemented in webimpress/phpunit-migration which I am using for updating PHPUnit.  Will push another commit with change to `self::` for assertions. 

Thanks, @Ocramius!